src/routes/profile/+page.svelte

```diff
@@ -37,20 +37,24 @@
   });
 </script>
 
-<div class="flex flex-col md:flex-row w-[90%] mx-auto space-y-4 md:space-y-0 md:space-x-4 mt-[-25px]">
-
-  <div class="flex-none w-full md:w-1/3 md:sticky md:top-4 h-max">
-    <ProfileInfo />
-  </div>
-
-  <div class="flex-1">
-    {#if loading}
-      <p>Loading projects...</p>
-    {:else if error}
-      <p>Error: {error}</p>
-    {:else}
-      <ProjectSection projects={userProjects} />
-    {/if}
+<!-- new code -->
+<!-- container below to center the inner container -->
+<div class="flex w-full items-start justify-center">
+  <div class="w-[90%] mx-auto grid justify-center md:flex md:flex-row space-y-4 md:space-y-0 md:space-x-4">
+    
+    <div class="px-5 md:px-0 md:w-1/3 bg-yellow-50 md:sticky h-max">
+      <ProfileInfo />
+    </div>
```

Comment:
```suggestion
    <div class="px-5 md:px-0 md:w-1/3  md:sticky h-max">
      <ProfileInfo />
    </div>
```

removing the `bg-yellow-50`